doc/api/player_events.md

```diff
@@ -326,7 +326,9 @@ properties:
 
     - ``frameRate`` (``string|undefined``): The video framerate.
 
-
+A null payload means that video track has been disabled.
+/!\ On Safari in _DirectFile_, a video track may have been disabled by user but
+still playing on screen.
```

Comment:
I may add the precision that we think that this is a Safari bug.
Also replace `/!\` by :warning: :p 
(actually  we could also use just the `⚠️` character, which is unicode)